Makefile

```diff
@@ -36,8 +36,9 @@ postgres:
 	docker run -d --name=postgres-dev --rm \
 		-e POSTGRES_PASSWORD=password123 \
 		--tmpfs=/var/lib/postgresql/data \
-		-p 15432:5432 \
-		postgres:14-alpine -c fsync=off -c full_page_writes=off
+		-p 127.0.0.1:15432:5432 \
+		postgres:14-alpine -c fsync=off -c full_page_writes=off \
+			-c listen_addresses=127.0.0.1 -c max_connections=100
```

Comment:
These are the default settings, but I'd like to include them here to make it easier to change them.

On linux setting `listen_addresses` to `0.0.0.0` allows tests to bypass the port mapping proxy and connect directly to the container.  Setting `max_connections` can be useful when testing concurrency.